### ISSUE TITLE
old config entities keep showing

### ISSUE THREAD

**Issue by Mariusthvdb** [gadgetchnnel/lovelace-home-feed-card#33]
even though the config has been changed, cache has been refreshed, and even the HA instance has been restarted...

this is on a Mac Safari, hassio 106.2
version 0.3.9 for home-feed-card

config:

```
  - type: 'custom:home-feed-card'
    title: Home Feed
    show_empty: false
    show_notification_title: true
#    compact_mode: true
    id_filter: rss-feed*
    more_info_on_tap: true

#    entities:
#        - entity: device_tracker.life360_marijn
#          name: Marijn
#          content_template: >
#            {{display_name}} arrived at {{state}} ({{latitude}},{{longitude}})
#          include_history: true
#          max_history: 5
#          remove_repeats: false
#          state_map:
#            not_home: Unknown Destination
```
the comments section keeps popping up in the frontend.

<img width="222" alt="Schermafbeelding 2020-03-02 om 14 05 43" src="https://user-images.githubusercontent.com/33354141/75678904-ee384d00-5c8e-11ea-8b73-673049f82fba.png">

**Comment by gadgetchnnel** [comment]
There was some code I had added to avoid trying to refresh history entities (entities with **include_history** set) if there were no entities in the config. Unfortunately this meant that, if you had history entities but then removed all entities, the old cached items will still remain.

I have fixed this in the latest beta (0.4.0b2).

**Comment by gadgetchnnel** [comment]
Closing this as this fix has now been released in 0.4.0
